tests/news_test.py

```diff
@@ -39,16 +47,77 @@ def __init__(self, *args, **kwargs):
         with (SAMPLE_PATH / "news_university_news_features_articles_2020.html").open() as f:
             self.university_news_features_articles_2020 = f.read()
 
+    @responses.activate
+    def test_get_categories(self):
+        news.get_categories.cache_clear()
+        news._scrape_categories.cache_clear()
+        responses.add(responses.GET, news.PITTWIRE_URL, body=self.pittwire)
+
+        categories = news.get_categories()
+
+        self.assertCountEqual(
+            categories, ["Features & Articles", "Accolades & Honors", "Ones to Watch", "Announcements and Updates"]
+        )
+
+    @responses.activate
+    def test_get_categories_missing(self):
+        news.get_categories.cache_clear()
+        news._scrape_categories.cache_clear()
+        responses.add(responses.GET, news.PITTWIRE_URL, body=self.pittwire_no_categories)
+
+        self.assertRaises(RuntimeError, news.get_categories)
+
+    @responses.activate
+    def test_get_topics(self):
+        news.get_topics.cache_clear()
+        news._scrape_topics.cache_clear()
+        responses.add(responses.GET, news.FEATURES_ARTICLES_URL, body=self.features_articles)
+
+        topics = news.get_topics()
+
+        self.assertCountEqual(
```

Comment:
Is this test going to hold true even into the future?

Reply:
Yes, all the tests for the `news` module are mocked. The mock files are in the `tests/samples` directory.